src/textual_inputs/text_input.py

```diff
@@ -151,6 +154,12 @@ def _render_text_with_cursor(self) -> List[Union[str, Tuple[str, Style]]]:
         """
         Produces the renderable Text object combining value and cursor
         """
+
+        self.cursor: Tuple[str, Style] = (
```

Comment:
Unless you anticipate the cursor attribute being accessed elsewhere we can make this a local variable.

```suggestion
        cursor: Tuple[str, Style] = (
```

Reply:
Fixed